tests/suite/cli_ui.rs

```diff
@@ -55,6 +55,9 @@ fn rustup_init_ui_doc_text_tests() {
         .unwrap();
 
         // Make sure that the help output of rustup-init and rustup-init.sh are the same.
-        assert_eq!(rustup_init_help_std_out, rustup_init_sh_help_std_out)
+        assert_eq!(
+            rustup_init_help_std_out.stdout.unwrap(),
+            rustup_init_sh_help_std_out.stdout.unwrap()
+        )
```

Comment:
I have no objection here, but I'm curious why the change

Reply:
Thats a great suggestion. Feel free to add it in incrementally where you need it.